VersionInfo.cs.in

```diff
@@ -10,13 +10,20 @@
 // General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
+#if DEBUG
+[assembly: AssemblyConfiguration("Debug")]
+#else
+[assembly: AssemblyConfiguration("Release")]
+#endif
+
+[assembly: AssemblyProduct("@K4A_PRODUCTNAME@")]
+[assembly: AssemblyCompany("@K4A_COMPANYNAME@")]
 [assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation. All rights reserved.")]
 [assembly: AssemblyTrademark("")]
 
 [assembly: CLSCompliant(true)]
 
 // This is the shared assembly version file for all of our assemblies.
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("@K4A_VERSION_MAJOR@.@K4A_VERSION_MINOR@.@K4A_VERSION_PATCH@.0")]
```

Comment:
Brent does have a K4A_VERSION_REVISION that he puts in instead of "0". Should this line follow that?

```suggestion
[assembly: AssemblyVersion("@K4A_VERSION_MAJOR@.@K4A_VERSION_MINOR@.@K4A_VERSION_PATCH@.@K4A_VERSION_REVISION@")]
```

Reply:
Ahh, I thought the revision was part of the build string. Since AssemblyVersion is used kind of as an identity of compatible binaries, I don't want that to specify the revision, instead I'll add it to the File and Informational Versions.